thirdeye/thirdeye-pinot/src/main/java/com/linkedin/thirdeye/detector/email/filter/PerformanceEvaluationUtil.java

```diff
@@ -0,0 +1,224 @@
+package com.linkedin.thirdeye.detector.email.filter;
+
+import com.linkedin.thirdeye.anomalydetection.context.AnomalyFeedback;
+import com.linkedin.thirdeye.constant.AnomalyFeedbackType;
+import com.linkedin.thirdeye.datalayer.dto.MergedAnomalyResultDTO;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
+import java.util.Properties;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+
+/**
+ * Utility class to evaluate the performance of a list of merged anomalies
+ */
+public class PerformanceEvaluationUtil {
```

Comment:
The name of the class is too generic. Why not call it XXXEvaluator. Util classes generally have a bunch of static methods.

What is the purpose of this class